src/playerphase.c

```diff
@@ -147,10 +150,14 @@ PROC_LABEL(3),
     PROC_WHILE(DoesBMXFADEExist),
 
     PROC_END,
-
 };
 
-struct ProcCmd CONST_DATA gProcScr_0859ACE8[] = {
+// clang-format on
+
```

Comment:
What was the on off for?

Reply:
I was wrapping each proc script individually with the toggles in case we wanted to move them around. But looking at it now it looks silly, so I removed the overuse of toggles in 5b43da4